src/CqlBuilderPanel/definitionsSection/definitions/Definitions.tsx

```diff
@@ -84,13 +84,17 @@ const Definitions = ({
     managePagination();
   }, [definitions, currentPage, currentLimit]);
 
-  const showEditDefinitionDialog = (index) => {
+  const populateReturnType = (index) => {
```

Comment:
not a big deal, but to keep the code readable, can we use verbs for method names? like

```suggestion
  const getPopulateReturnType = (index) => {
```

Reply:
Now that I think about it, it's not a population at all, it is a CQL definition

Reply:
I meant populate as a verb, somehow forgot the word "get"